MapboxCoreNavigation/SpokenInstructionFormatter.swift

```diff
@@ -0,0 +1,74 @@
+import Foundation
```

Comment:
I'd probably assume that this class extends NSFormatter but it doesn't need any of its functionality. I'd keep it as it is.